manifests/plugin/dynflow.pp

```diff
@@ -70,5 +73,14 @@
       ensure => running,
       enable => true,
     }
+    if $open_file_limit {
```

Comment:
If we have a default, then this if statement makes no sense anymore. If you remove the ```if```, you can change the notify to ```~>```